lib/services/verification.service.js

```diff
@@ -23,11 +22,15 @@ const getDocumentLoader = (didDoc) => {
                 contextUrl: null, documentUrl: url, document: DOCUMENTS[url]
             }
         }
-        return await jsonld.documentLoaders.node()(url);
-    });
+        let document = await (await fetch(url)).json();
```

Comment:
Facing following error with jsonld node document loader:
jsonld.InvalidUrl: Dereferencing a URL did not result in a valid JSON-LD object. Possible causes are an inaccessible URL perhaps due to a same-origin policy (ensure the server uses CORS if you are using client-side JavaScript), too many redirects, a non-JSON response, or more than one HTTP Link Header was provided for a remote context.

So, updated the part with fetch

Reply:
Did you try without this change and with all other changes?